composer.json

```diff
@@ -16,6 +16,9 @@
         "mikehaertl/php-tmpfile": "^1.0.0",
         "mikehaertl/php-shellcommand": "^1.0.2"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^6.0"
+    },
```

Comment:
Please remove this. We don't want every user to get phpunit installed with this package by default.

Reply:
ok, no prob... but it's good practice. It's in the "require-dev" so if user don't want it, he can do
```
composer install --no-dev
```